### PR TITLE
developer task

### PR DESCRIPTION
**Pull request by panicnot**
#### What does this PR do?

Dev task for interviews. Please read over it, check you would be happy for it to be sent out to a candidate 

---
#### Quality checklist (for PR author to complete BEFORE code review):
- [x] I am ready for this to be code reviewed, merged and tested.
#### Reviewer 1
- [x] +1 from me!
#### Reviewer 2
- [ ] +1 from me!

### CODE REVIEW

**Comment by pmcnr-hx** [comment]
Can this be group-wide rather than SB specific? HX should be offering this as an alternative to the front-end oriented Flickr API task for candidates who are more comfortable with back-end tech.

**Comment by panicnot** [comment]
sure, i'l move it

File: recruitment/developer-API-task.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+###Web Development Task
+
+You've been sent this task as we would love to get an idea of the way you work.
+
+At Holiday Extras we use a variety of different technologies and we don't expect you to be aware of all of them! We've designed this task with this in mind. It's worth noting that everything described below is a rough guideline of what we're looking for so don't feel you have to follow it exactly.
+
+
+#####Your task is to create an api to manage a user persistence layer.
```
Comment (pmcnr-hx) [inline recruitment/developer-API-task.md]: `API`, being an acronym, in upper case? (also other uses below)

File: recruitment/developer-API-task.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+Although the main outcomes of the task are listed above, if you feel like you want to go that extra mile and show us what you're capable of, here is a list of potential enhancements that we have come up with:
+
+
+* How your api is to be consumed (a custom interface or something like Google Chromes "Postman" or Swagger).
+* Use of an industry standard data exchange format.
+
+Alternatively if you can think of any other features that you feel would further enhance your api, then we'd love to see what you can come up with!
+
+When you feel you've completed the task we would appreciate it if you could publish it online so we are in a position to be able to review it. We recommend that you upload your code to a Github/Bitbucket repository, however it's fine if you'd rather send it as an attachment in an email.
```
Comment (pmcnr-hx) [inline recruitment/developer-API-task.md]: We can maybe suggest [GitLab](https://about.gitlab.com/) as another alternative to GitHub / Bitbucket.

File: recruitment/developer-API-task.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+	**created** - *The date and time the user was added*
+
+* It must have the ability to persist user information for at least the lifetime of the test.
+
+* You api must expose functionality to create, read, update and delete (CRUD) models.
+
+Although the main outcomes of the task are listed above, if you feel like you want to go that extra mile and show us what you're capable of, here is a list of potential enhancements that we have come up with:
+
+
+* How your api is to be consumed (a custom interface or something like Google Chromes "Postman" or Swagger).
```
Comment (pmcnr-hx) [inline recruitment/developer-API-task.md]: `Chromes` → `Chrome's`

Comment (pmcnr-hx) [inline recruitment/developer-API-task.md]: Suggest a client consumer library also as an alternative?

**Comment by pmcnr-hx** [comment]
Just some minor comments, nothing fundamental, so good to go from me: :+1:

**Comment by notABluesSinger** [comment]
Agree with all of @pmcnr-hx's comments, other than that :+1:

**Pull request merged**